src/clientSideScene/sceneEntities.ts

```diff
@@ -616,7 +615,7 @@ export class SceneEntities {
 
         const inserted = insertNewStartProfileAt(
           kclManager.ast,
-          sketchDetails.sketchEntryNodePath,
+          sketchDetails.sketchEntryNodePath || [],
```

Comment:
found out this function more or less expects an empty sktechEntryNodePath for cases where there are no profiles yet.